roles/cibox-jenkins/defaults/main.yml

```diff
@@ -1,5 +1,9 @@
 ---
 placeholder: 'placeholder'
+project_name: 'cibox'
+jenkins_admin_email: 'admin@cibox.tools'
+# Jenkins server IP. Should be overridden outside of role.
+jenkins_server_ip: '192.168.1.1'
```

Comment:
I think we can pass one additional variable into `extra_vars` in bash script and solve this https://github.com/propeoplemd/cibox/issues/326